kimchi/src/circuits/polynomials/range_check/gate.rs

```diff
@@ -76,21 +38,9 @@ impl<F: PrimeField> CircuitGate<F> {
     ///       next_row      - next row after this gate
     ///       circuit_gates - vector of circuit gates comprising this gate
     pub fn create_multi_range_check(start_row: usize) -> (usize, Vec<Self>) {
-        let mut wires: Vec<GateWires> = (0..4).map(|i| Wire::new(start_row + i)).collect();
+        let wires: Vec<GateWires> = (0..4).map(|i| Wire::new(start_row + i)).collect();
```

Comment:
can you inline this within each CircuitGate? Instead of creating this here in advance